benchmark/cluster_containers_setup.py

```diff
@@ -129,8 +129,8 @@ def main(reactor, argv, environ):
     # output
     eliot_to_stdout(MESSAGE_FORMATS, ACTION_START_FORMATS)
 
+    options = ContainerOptions()
     try:
-        options = ContainerOptions()
```

Comment:
:question: Looks like a good change, but I'm curious how you noticed it.
My pylint doesn't highlight this problem.